pkgs/applications/networking/cluster/helm2/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildGoModule, fetchFromGitHub, installShellFiles, Security }:
+
+buildGoModule rec {
+  pname = "helm";
+  version = "2.16.3";
+
+  src = fetchFromGitHub {
+    owner = "helm";
+    repo = "helm";
+    rev = "v${version}";
+    sha256 = "16hbwmgq14g28r9s0ipnpiqlppyh57yrcqcspmj05vrf9jsg5vwj";
+  };
+  modSha256 = "0618zzi4x37ahsrazsr82anghhfva8yaryzb3p5d737p3ixbiyv8";
+
+  subPackages = [ "cmd/helm" ];
+  buildFlagsArray = [ "-ldflags=-w -s -X helm.sh/helm/v3/internal/version.version=v${version}" ];
```

Comment:
I believe it should be different for helm v2 - https://github.com/helm/helm/blob/release-2.16/versioning.mk#L25-L26

Reply:
I've ported this one from the nixpkgs git history. Helm 2 was built this way before, but it probably makes sense to update it as well as the version.

Reply:
I've tested and seems like it works correctly, so, we can keep as it was before.